test/GithubApi.Gist.test.js

```diff
@@ -0,0 +1,38 @@
+const agent = require('superagent-promise')(require('superagent'), Promise);
+const chai = require('chai');
+const statusCode = require('http-status-codes');
+
+const { expect } = chai;
+
+describe('Given a DELETE tests', () => {
```

Comment:
para que el código quede más claro utiliza los before para todas las instancias y los it solo para los expects

Reply:
No entendi, pero coloque la definición del archivo por fuera para hacer el código más claro

Reply:
esto en un before
```
agent.post('https://api.github.com/gists', gistToCreate)
  .auth('token', process.env.ACCESS_TOKEN).then((response) => {
    gist = response.body;
}));
```
esto en un it
```
 expect(response.status).to.equal(statusCode.CREATED);
 expect(gist).to.containSubset(gistToCreate);
```

Reply:
No puedo hacerlo, ya que debo en la consulta del post verificar que el status sea creado y además asignar el body a la variable gist

Reply:
también puedes guardar el response completo, no solo el body

Reply:
Listo :)